config/content.d/support.rackspace.com.json

```diff
@@ -8,6 +8,9 @@
         },
         "proxy": {
             "/favicon.ico": "https://8d8dcdd952aa2708c2ff-519cda130c91226e76017ae910bdb276.ssl.cf1.rackcdn.com/favicon-aa186bee158ecea4c9b6a98e2ceec3141b6b7d8d94dcb71731ba112e2b17b1db.ico"
+        },
+        "proxy": {
+            "/images/favicon.ico": "https://8d8dcdd952aa2708c2ff-519cda130c91226e76017ae910bdb276.ssl.cf1.rackcdn.com/favicon-aa186bee158ecea4c9b6a98e2ceec3141b6b7d8d94dcb71731ba112e2b17b1db.ico"
```

Comment:
These should be two keys within the `proxy` object:

```json
{
  "proxy": {
    "/favicon.ico": "...",
    "/images/favicon.ico": "..."
  }
}
```

Otherwise one will randomly overwrite the other :sparkles: